english translation, combined with Eeko's pull request

lokori opened the pull request:
This translation is a modified version of one submitted in the pull request 22: https://github.com/kybervpk/website/pull/22

This combines my earlier (non published) translation with the one submitted by @Eeko in that other pull request and in my opinion this is better than either one. We had a lot of similarities as well in our translations.

anttivi-solita committed:
english translation, combined with one from Pull request https://github.com/kybervpk/website/pull/22

lokori commented:
I submitted another pull request at https://github.com/kybervpk/website/pull/23. I had written a translation yesterday and it was under review. I combined both translations to that pull request and definitely some things were much better in this version than in my original version.

anttivi-solita committed:
fixed some typos

anttivi-solita committed:
times to 24h format

Eeko commented on checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+*   Was it your work email instead of your personal email? Contact your employer. For example, talk with your human resources manager. You may get support.
+*   Consider special [non-disclosure for personal safety](https://dvv.fi/en/non-disclosure-for-personal-safety). This is a special protection for cases with a serious threat of physical violence. Talk with the police.
+*   Monitor your digital footprint actively in the future. You can use [Have I been pwned?](https://haveibeenpwned.com/)  -service to notify you if your information has been leaked publicly. The service is maintained by the security expert Troy Hunt. Alternatively, you can use the [Identity Theft Checker from Finnish F-Secure](https://www.f-secure.com/fi/home/free-tools/identity-theft-checker) or ...
```
url of Badrap.io should be https://badrap.io/ ,

Eeko commented on checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+### City of Helsinki provides help for residents around the clock
+
+*   Telephone health advice phone [09 10023](tel:+358910023).
+*   Crisis Emergency Support of the City of Helsink phone. [09 310 44222](tel:+358931044222).
```
Should be "City of Helsinki", not "City of Helsink".

Eeko commented on checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+---
+title: "Checklist for the victims of a data breach"
+description: We collected a comprehensive list of notes especially for the victims of the Vastaamo data breach. If you have been a client of Vastaamo before the year 2018, your confidential information may have been stolen. According to Vastaamo, copies and accessed customer records might exist from early 2019, likely until March 2019.
+---
```
Should include an "aliases" section according to the website syntax?

lokori replied:
Possibly, I'm not certain.

Eeko reviewed (changes requested):
File should be moved to website/content/en/releases?

anttivi-solita committed:
fixed minor typos

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+---
+title: "Checklist for the victims of a data breach"
+description: We collected a comprehensive list of notes especially for the victims of the Vastaamo data breach. If you have been a client of Vastaamo before the year 2018, your confidential information may have been stolen. According to Vastaamo, copies and accessed customer records might exist from early 2019, likely until March 2019.
```
We have collected a comprehensive list of information for the victims of the Vastaamo data breach. If you have been a client of Vastaamo your confidential information may have been stolen.

Current information about the data breach can be found at Vastaamo's web pages

lokori replied:
This is better. I added "before March 2019" because this is not related to recent customer relationships.

iolivergithub replied:
Careful with the dates...probably best to make it simple for the reader, even if it does mean Vastaamo get more queries.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+You can follow Vastaamo's official updates [at their website](https://vastaamo.fi/ajankohtaista/). (currently only in Finnish)
+
+## Presumably, the stolen data contains the following:
```
Drop the word "presumably".  

The stolen data contains the following information:

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+All of the above are classified as [personal information](https://tietosuoja.fi/en/what-is-personal-data). It is possible that your information has not been stolen or that only portions of it have leaked.
+
+## If you suspect you have been a victim of the breach or you have received a personal ransom request, follow these steps:
```
If you suspect you have been a victim of the breach or you have received a personal ransom request, please follow these steps:

lokori replied:
Yes, it's more polite :) Added that little word.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+### Keep a diary of your actions
+
+*   Write down your actions. What you do, when and to whom. For example: _Filed a criminal complaint with the police on 24th of October 2020 at 10:05, requested prohibition of credit from Asiakastieto 24th of October at 5pm._
```
Write down your actions:   What you do, when and to whom.   For example: ....

iolivergithub replied:
Change to:



Write down your actions: what you do, when and to whom. Also include how much time it has taken you. For example: ....

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+### Collect evidence
+
+*   If you have received a ransom blackmail, take a screenshot of it and store it on your computer for investigation. Store the original message as well.
```
Comment:  is the ransom being sent by email only?

lokori replied:
As far as I know, yes. This may change at some point.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+*   Screenshots are important evidence for the authorities. The screenshot should include the sender address.
+*   How to take screenshots in Windows 10: [Instructions from Microsoft](https://support.microsoft.com/en-us/windows/open-snipping-tool-and-take-a-screenshot-a35ac9ff-4a58-24c9-3253-f12bac9f9d44)
+*   How to take screenshots in Apple macOS: [Instructions from Apple](https://support.apple.com/en-us/HT201361)
+*   You can also take a photo of your computer's screen. Make sure that the text is readable from the photo.
```
You can also take a photograph of your computer's screen using a camera. You can use your mobile phone's camera for this. Make sure that the text is readable in the photo. If it helps you can take multiple pictures to cover the whole screen in enough detail.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+*   How to take screenshots in Windows 10: [Instructions from Microsoft](https://support.microsoft.com/en-us/windows/open-snipping-tool-and-take-a-screenshot-a35ac9ff-4a58-24c9-3253-f12bac9f9d44)
+*   How to take screenshots in Apple macOS: [Instructions from Apple](https://support.apple.com/en-us/HT201361)
+*   You can also take a photo of your computer's screen. Make sure that the text is readable from the photo.
+*   Keep receipts from all incurred expenses. For example, services related to credit prohibitions.
```
Kepp receipts from all incurred expenses. For example, if you have had to call a credit agency, bank, credit card handler etc.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+### Do not pay the ransom or communicate with the blackmailer
+
+*   The extortion messages contain one's name and identification number. Similar messages have been sent to thousands of other people as well.
```
The extortion message(s) will contain your name and identification number. Similar messages have been sent to potentially thousands of other people as well.

lokori replied:
we know that they have been sent to many people so it's not speculative, though we don't know the exact number.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+### Do not pay the ransom or communicate with the blackmailer
+
+*   The extortion messages contain one's name and identification number. Similar messages have been sent to thousands of other people as well.
+*   Paying the ransom funds organized crime and the criminals may extort you again or alternatively, publish your information despite payment. Do not communicate with the criminal. The information has most likely been leaked to other parties already.
```
Paying the ransom funds organized crime and the criminals may try extort you again or even try to publish your information despite payment. Under no circumstances should you communicate with the extorters. Keep a record as described above. This information will be valuable to the Police for their investigations.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+*   The extortion messages contain one's name and identification number. Similar messages have been sent to thousands of other people as well.
+*   Paying the ransom funds organized crime and the criminals may extort you again or alternatively, publish your information despite payment. Do not communicate with the criminal. The information has most likely been leaked to other parties already.
+
+### File a criminal complaint with the police for data leak and/or extortion
```
### File a Criminal Complaint with the Police

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+*   Paying the ransom funds organized crime and the criminals may extort you again or alternatively, publish your information despite payment. Do not communicate with the criminal. The information has most likely been leaked to other parties already.
+
+### File a criminal complaint with the police for data leak and/or extortion
+
```
Extra:   Finnish Police have a dedicated unit for dealing with cyber crimes of this nature.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+*   You can report an offence online on the [website of the Finnish Police](https://www.poliisi.fi/crimes/reporting_an_offence_online). The service is available every day from 6 to 22:45. If you are not able to do a report immediately, try again later. If you have received a ransom note, include a screenshot of it as an attachment.
+*   If you can't report an offence online, do it at your local police station.
+*   Police has instructed to not call the Emergency Response Centers.
```
The Finnish Police have requested that you do not call the Emergency Repsonse Centers (eg: by dialling 112). If you have a query then contact the Finnish Police at the website given above.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+*   The extortion messages contain one's name and identification number. Similar messages have been sent to thousands of other people as well.
+*   Paying the ransom funds organized crime and the criminals may extort you again or alternatively, publish your information despite payment. Do not communicate with the criminal. The information has most likely been leaked to other parties already.
+
```
An additional point - people reading this in English are likely to be very unfamiliar with the Finnish Police, so:

There are two points of contact with the authorities: The Finnish Polices and the National Center for Cyber Security. Below are the instructions for contacting both of these authorities.  If you have received a crime number from the Finnish Police then include this number in any communications with both authorities.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+### File a criminal complaint with the police for data leak and/or extortion
+
+*   You can report an offence online on the [website of the Finnish Police](https://www.poliisi.fi/crimes/reporting_an_offence_online). The service is available every day from 6 to 22:45. If you are not able to do a report immediately, try again later. If you have received a ransom note, include a screenshot of it as an attachment.
+*   If you can't report an offence online, do it at your local police station.
```
If you can not report the office online please visit your nearest police station.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+### Report the incident to National Cyber Security Centre 
+
+*   You can file a report with the [form provided by Finnish Cyber Security Centre](https://www.kyberturvallisuuskeskus.fi/en/report)
+*   Based on the notifications, the NCSC-FI investigates security violations and threats to information security on network services, communications services and added value services, collects information on these incidents, and helps organisations with instructions and sometimes with technical assistance.
```
Based on these notifications from you, the NCSC-FI can investigate security violations and threats to information security, communications services etc. The information about these incidents is collected and helps organisations with instructions and sometimes with technical assistance.

anttivi-solita committed:
fixed a few things based on pull request comments

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+*   This ensures that nobody can take a loan in your name. Finland has two companies maintaining credit registries: Asiakastieto and Bisnode. We recommend you request for a credit ban from both. Note that the credit ban lasts for two years at a time and you will have to renew it if you wish to continue it.
+*   Asiakastieto will charge 19,95€ for the ban. [You can request it online from Asiakastieto](https://www.asiakastieto.fi/omatieto/en/tuotteet/oma-luottokielto) (currently only in Finnish)
+*   Bisnode credit ban costs 15,95€. [You can request it online from Bisnode](https://my.bisnode.fi/protection) (currently only in Finnish)
+*   A credit ban can prevent frauds using your personal information. Using the personal identification number and other personal information, a criminal may be able to make small purchases or open other lines of credit in your name. This voluntary credit ban does not prevent you from buying in installments or requesting other credit. You'll receive a separate certificate, which you can personally ...
```
A credit ban can prevent criminals from using your personal information and comitting further fraud. Using the personal identification number and other personal information, a criminal may be able to make small purchases or open other lines of credit in your name. This voluntary credit ban does not prevent you from buying in installments or requesting other credit. You will receive a separate certificate which you can use to prove your creditworthiness despite the voluntary credit ban.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+*   You can prevent address changes by activating [address change protection from Posti](https://www.posti.fi/fi/asiakastuki/tiedotteet/postin-ohje-psykoterapiakeskus-vastaamon-tietomurtoon-liittyen-nain-teet). (currently only in Finnish)
+*   The ban is requested from [Posti's online customer service portal](https://www.posti.fi/omatpalvelut/feedback?lang=fi) under "Address change protection, Feedback about Mail forwarding or Nearby Mailbox."
+*   The protection prevents address changes and mail delivery suspensions as long as the protection is enabled. A criminal can't then order deliveries or invoices under your name to another address.
```
The protection prevents address changes and mail delivery suspensions as long as the protection is enabled. A criminal can not then order deliveries or invoices under your name to another address.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+### Request a registration ban from Finnish Patent and Registration Office (PRH) free-of-charge
+
+*   A registration ban will prevent fraud where you are registered as a responsible person for some company. When you request a registration prohibition from PRH, you can't be registered as responsible for any company or registered organization.
```
A registration ban will prevent fraud where you are registered as a responsible person for some company. When you request a registration prohibition from PRH, you can not be registered as responsible for any company or registered organization.

anttivi-solita committed:
fix typos

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+*   Phishing and spam emails may increase and several parties my try to extort you. Cyber criminals may distribute and sell your information to other criminals and some may try guessing your passwords on different services. Make sure you have a strong password in all services and start using two-factor authentication if possible. You should also change the password of the email account which has  ...
+*   Instructions for creating strong passwords
+    1. The longer your password is, the safer it is. Good minimum length is 16 characters.
```
Just:   The longer your password is, the safer it is.  We would recommend including a mix of numbers, upper and lower case letters and a symbol. 

Length of password ---->  difficult one,,  16 characters would be too much for many people

iolivergithub replied:
OK, I just combined #2 with #1 .. split accordinly

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+    1. The longer your password is, the safer it is. Good minimum length is 16 characters.
+    2. Use both small and capital letters combined with special and numeric characters.
+    3. Typos, dialects, spoken idioms and other "broken words" make your password stronger.
+    4. Use a unique password for every service.
```
Try to use a different password for each account and service you use

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+    2. Use both small and capital letters combined with special and numeric characters.
+    3. Typos, dialects, spoken idioms and other "broken words" make your password stronger.
+    4. Use a unique password for every service.
+    5. Be especially careful with important passwords, like the email account passwort that can be used to reset other passwords.
```
spelling:   like the email account password that can be used to reset other passwords

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+    3. Typos, dialects, spoken idioms and other "broken words" make your password stronger.
+    4. Use a unique password for every service.
+    5. Be especially careful with important passwords, like the email account passwort that can be used to reset other passwords.
+    6. Never reveal your passwords to anyone! Official authorities will never ask you to disclose your password!
```
Never reveal your passwords to anyone! Official authorities, banks etc will never ask you to disclose your password!  If you are every asked for the password decline to give it: there will never be any need to tell anyone your password.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+### Check your insurance policy
+
+*   Many home insurances provide limited legal assistance for victims of an identity fraud. The data privacy legislation may give you a right for damage compensation.
```
Many home insurance policies provide limited legal assistance for victims of identity fraud. The Finnish data privacy legislation (citation?!) may give you further rights to compensation.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+*   According to European General Data Protection Regulation (GDPR), you have the right to know what data Vastaamo has collected about you. This can help you decide what you want to do next.
+*   Contact Vastaamo with their [request for access to personal data](https://vastaamo.fi/files/rekisteritietojen_tarkastuspyynto.pdf) -form. More information about Vastaamo Data Protection is available from [their website](https://vastaamo.fi/tietosuoja/). (currently only in Finnish)
+*   Understand that it may take several weeks to get an answer.
```
It may take up to several weeks to obtain an answer. However, by EU Law, the company must comply with a request for information

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+*   Many home insurances provide limited legal assistance for victims of an identity fraud. The data privacy legislation may give you a right for damage compensation.
+
+### Make a request to access your personal data from Vastaamo
+
```
This is called a "Subject Access Request"

iolivergithub replied:
It is a right enshrined in EU and national laws.

lokori replied:
good point, fixed

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+### Take care of yourself
+
+*   Don't panic. It's possible that only some of your data has been leaked.
```
I would move this to the top.

Don't Panic
While it is possible that some of your data may have been leaked, there is support and advice available.

lokori replied:
This is not a bad idea. I would suggest that we do it also for the Finnish version if we do that.

lokori replied:
It has been moved up in both versions now.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+*   Was it your work email instead of your personal email? Contact your employer. For example, talk with your human resources manager. You may get support.
+*   Consider special [non-disclosure for personal safety](https://dvv.fi/en/non-disclosure-for-personal-safety). This is a special protection for cases with a serious threat of physical violence. Talk with the police.
+*   Monitor your digital footprint actively in the future. You can use [Have I been pwned?](https://haveibeenpwned.com/)  -service to notify you if your information has been leaked publicly. The service is maintained by the security expert Troy Hunt. Alternatively, you can use the [Identity Theft Checker from Finnish F-Secure](https://www.f-secure.com/fi/home/free-tools/identity-theft-checker) or ...
```
Are haveibeenpwned now offering a data breach notification .... IIRC it is just notification that your email/login details have been leaked - not necessarily the data associated with that account.

lokori replied:
Afaik it depends on the case, but it's just the email/login hash mostly.  The notification works based on email address,but they tell what other information may have been associated with it in the leak.

iolivergithub commented on content/en/releases/checklist-for-victims-of-a-data-breach.md:
```diff
@@ -0,0 +1,187 @@
+
+### Vastaamo crisis phone
+
+*   Contact number: [044 4141 997](tel:+358444141997), open on Monday to Friday 21, Saturday 10 to 14, 17 to 21 and Sunday 10 to 21
```
Languages availble?   Finnish, Swedish, English presumably?

lokori replied:
Presumably. We don't actually know for certain.

anttivi-solita committed:
minor fixes based on comments

shadikka committed:
feat: Add OpenGraph title and description tags (+fix minor whitespace
issue)

shadikka committed:
content: Guidance for ID theft victims v2.0

appelsiini committed:
Merge pull request #25 from kybervpk/more-meta

Add OpenGraph title and description tags

Jaroneko committed:
Merge pull request #26 from kybervpk/id-theft-2.0

Guidance for ID theft victims v2.0

shadikka committed:
fix: Update Elisa number

The originally included number was a paid number for more hands-on guidance; change it to the free general customer service one.

Jaroneko committed:
Merge pull request #27 from kybervpk/elisa-fix

fix: Update Elisa number

Jaroneko committed:
Removed Marko from roster

shadikka committed:
Merge pull request #28 from Jaroneko/master

Removed Marko from roster

anttivi-solita committed:
updates based on updates on the Finnish version 26.10.

Jaroneko committed:
Merge branch 'master' of https://github.com/lokori/website into master

Jaroneko committed:
Modified language slightly and added new bits from the Finnish 2.0 version. Still missing Klarna and phone operators.

anttivi-solita committed:
minor fix based on comments

Eeko commented:
You're free to use my translation texts and license them as you see fit.

anttivi-solita committed:
Added paragraph about Klarna and teleoperators

anttivi-solita committed:
minor fixes

anttivi-solita committed:
minor fixes

Jaroneko reviewed (approved):
Looks good for release
